portfolio/src/main/webapp/index.html

```diff
@@ -7,7 +7,19 @@
     <script src = "script.js"></script>
   </head>
   <body>
-      
+
+        <div class = "container navbar">
```

Comment:
Should this be aligning with the `div` on line 21?

Reply:
Yes, I fixed this